src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -22,50 +22,66 @@
 Yast.import "UI"
 
 module UpdateAlternatives
-
+  # List all alternatives for an alternative group.
```

Comment:
ACtually this class is not list all alternatives, it is dialog. It should write something like `Dialog for displaying possible alternatives for one particular group.`?